tests/test_utilities.py

```diff
@@ -156,6 +156,132 @@ def test_check_all_contiguous():
     t2 = time.time()
     print 'time for %s = %.2f'%(funcname(),t2-t1)
 
+def test_interleaveImages():
+    import time
+    t1 = time.time()
+    # 1) Interleave arrays as Images
+    x_size, y_size = 16,10
+    n1, n2 = 2, 5
+    im_list = []
+    for i in xrange(n1):
+        for j in xrange(n2):
+            x = np.arange(-x_size+2.*i/n1,x_size+2.*i/n1,2)
```

Comment:
No particular reason. This test is now removed completely since ``offsets`` is made mandatory now.